### PR TITLE
SSB: Add EpicNikolai

### PR DESCRIPTION
**Pull request by Volco**

### CODE REVIEW

File: data/mods/ssb/abilities.ts [hunk 1]
```diff
@@ -295,6 +295,20 @@ export const BattleAbilities: {[k: string]: ModdedAbilityData} = {
 		},
 	},
 
+	// EpikNikolai
+	dragonheart: {
+		desc: "Once per battle, at 25% or lower this pokemon heals 50% hp.",
+		shortDesc: "Heals 50% when 25% or lower once per battle.",
+		name: "Dragon Heart",
+		onDamage(damage, target, source, move) {
+			if (target.heartless) return;
```
Comment (ghost) [inline data/mods/ssb/abilities.ts]:
```suggestion
			if (target.m.heartless) return;
```

File: data/mods/ssb/abilities.ts [hunk 2]
```diff
@@ -295,6 +295,20 @@ export const BattleAbilities: {[k: string]: ModdedAbilityData} = {
...
+			if (target.hp <= target.maxhp / 4) {
+				this.heal(target.baseMaxhp / 2);
+				target.heartless = true;
```
Comment (ghost) [inline data/mods/ssb/abilities.ts]:
```suggestion
				target.m.heartless = true;
```

**Comment by ghost** [comment]
Also, the username is Epi**c**Nikolai, not Epi**k**Nikolai. Besides that looks to go!

**Comment by Volco** [comment]
remind me not to do stuff when tired as shit because i read it as k instead of c lmao

File: data/mods/ssb/statuses.ts [hunk 3]
```diff
@@ -256,6 +256,18 @@ export const BattleStatuses: {[k: string]: ModdedPureEffectData} = {
 			this.add(`c|${getName('Emeri')}|don't forget to chall SFG or Agarica in gen8ou`);
 		},
 	},
+	epiknikolai: {
+		noCopy: true,
+		onStart() {
+			this.add(`c|${getName('EpikNikolai')}|I never give up until I get something right, which means destroying you ☜(ﾟヮﾟ☜)`);
```
Comment (ghost) [inline data/mods/ssb/statuses.ts]:
```suggestion
			this.add(`c|${getName('EpikNikolai')}|I never give up until I get something right, which means destroying you ☜(ﾟヮﾟ☜)`);
			if (source.species.id !== 'garchompmega' || source.illusion) return;
			this.add('-start', source, 'typechange', source.types.join('/'), '[silent]');
```

**Comment by Volco** [comment]
note to future self that i will definitely remember and/or follow (yeah right) don't code at 3 or 4 am after being up for almost 24 hours lol

**Comment by ghost** [comment]
If only

**Pull request merged**